WordPress/WordPressUITests/Tests/SupportScreenTests.swift

```diff
@@ -25,4 +25,19 @@ class SupportScreenTests: XCTestCase {
         //Dismiss because tearDown() can't handle modals currently
         supportScreen.dismiss()
     }
+
+    func testContactUsCanBeLoadedDuringLogin() throws {
+        let contactUsScreen = try PrologueScreen()
+            .selectContinue()
+            .selectHelp()
+            .contactSupport()
+
+        XCTAssert(contactUsScreen.isLoaded)
+        XCTAssert(contactUsScreen.canSendMessage())
```

Comment:
It's subjective, but if there was a dedicated `assertContactUsScreenLoaded` method, returning `ContactUsScreen`, and also having `canSendMessage` separated into smaller methods, it would be possible to fully use the advantage(?) of method chaining and have a test looking something like this:

```suggestion
    func testContactUsCanBeLoadedDuringLogin() throws {
        PrologueScreen()
            .selectContinue()
            .selectHelp()
            .contactSupport()
            .assertContactUsScreenLoaded()
            .enterText("A")
            .assertCanSendMessage()
```

(just a thought, it just about cosmetic change)